bin/kano-settings-onboot

```diff
@@ -429,13 +472,22 @@ if __name__ == '__main__':
         set_to_HDMI(False)
 
     # calculate changes needed
-    changes = compare_and_set_mode() or \
-        compare_and_set_full_range() or \
-        compare_and_set_overscan() or \
-        compare_and_set_optimal_resolution()
+    overall_changes = False
 
-    if changes:
+    changes = compare_and_set_mode(edid, status)
+    overall_changes = overall_changes or changes
+
+    changes = compare_and_set_full_range(edid, status)
+    overall_changes = overall_changes or changes
 
+    changes = compare_and_set_overscan(edid, status)
+    overall_changes = overall_changes or changes
+
+    changes = set_resolution_for_screen_kit(supported) or \
+              compare_and_set_optimal_resolution(edid, status, supported, verbose)
+    overall_changes = overall_changes or changes
+
+    if changes:
```

Comment:
No, you're absolutely right.. Obviously that's what those lines were working up towards..

Good spot, thanks 👍